function.js

```diff
@@ -0,0 +1,24 @@
+var counter = 0;
```

Comment:
Great job on refactoring. Really well done and cleanly written.

Remind me to help you with the Sublime settings for indentation.